values.yaml

```diff
@@ -67,7 +67,21 @@ st2:
   config: |
     [api]
     allow_origin = '*'
-
+  
+  #Override Definitions can be added here. 
+  #  overrides:
+  #    _global.yaml: |
+  #    ---
+  #    rules:
+  #      defaults:
+  #        enabled: true
+  #  packA.yaml: |
```

Comment:
Looks like a wrong indentation here for `packA.yaml`, compared to `_global.yaml`.

Reply:
The indentation should be fixed now.